### PR TITLE
Algolia Search Documentation

### PR DESCRIPTION
**Pull request by u12206050**
Basic example of including Algolia search.

### CODE REVIEW

File: docs/guide-search.md [hunk 1]
```diff
@@ -1,3 +1,127 @@
 # Add search to Gridsome
 
-_Contribute..._
\ No newline at end of file
+## Using [Algolia](https://www.algolia.com/)
+
+Currently Algolia offers upto 50,000 operations for free which should be enough for any site with low traffic. Especially if search isn't used as often.
```
Comment (ahtee) [inline docs/guide-search.md]: typo on `upto`

File: docs/guide-search.md [hunk 2]
```diff
@@ -1,3 +1,127 @@
...
+
+### Setup
+
+  1. Create a free Algolia account
+  2. Install [gridsome-plugin-algolia](https://gridsome.org/plugins/gridsome-plugin-algolia) eg. `npm install gridsome-plugin-algolia -S`
```
Comment (ahtee) [inline docs/guide-search.md]: depending on how the rest of projects document their plugins, commonly I see this command format as:

```
npm install --save gridsome-plugin-algolia 
```

File: docs/guide-search.md [hunk 3]
```diff
@@ -1,3 +1,127 @@
...
+  3. Follow the setup instructions on the plugin page for setting up the Algolia indexing process.
+  4. Create the ui using `algoliasearch` and `vue-instantsearch`, run `npm install algoliasearch vue-instantsearch -S`
```
Comment (ahtee) [inline docs/guide-search.md]: capitalize UI

**Comment by thetre97** [comment]
Also closes #564

**Review by thetre97 — APPROVED** [review]

**Pull request merged**